src/modules/auth/controllers/authentication.controller.ts

```diff
@@ -0,0 +1,18 @@
+import { Controller, Post, Body, UseGuards } from '@nestjs/common';
+import { AuthenticationService } from '@services/authentication.service';
+import { LoginDto, RefreshDto } from '@dtos/auth.dto';
+import { JwtAuthGuard } from '@guards/jwt-auth.guard';
+import { Public } from '@decorators/public-route.decorator';
+import { PasetoAuthGuard } from '@guards/paseto-auth.guard';
+
+@Controller('auth')
+export class AuthenticationController {
+  constructor(private readonly authService: AuthenticationService) {}
+  
+  // @Public()
```

Comment:
Why comment this decorator?

Reply:
maybe when doing the task, i assume refresh token is in headers so it is not a public route

Reply:
Each token has a set expiry. If we dont make this route public, the token strategy will reject and return 401